crates/arroyo-connectors/src/iggy/table.json

```diff
@@ -0,0 +1,86 @@
+{
+    "type": "object",
+    "title": "IggyTable",
+    "properties": {
+        "stream": {
+            "title": "Stream",
+            "type": "string",
+            "description": "The Iggy stream to use for this table"
+        },
+        "topic": {
+            "title": "Topic",
+            "type": "string",
+            "description": "The Iggy topic to use for this table"
+        },
+        "type": {
+            "type": "object",
+            "title": "Table Type",
+            "oneOf": [
+                {
+                    "type": "object",
+                    "title": "Source",
+                    "properties": {
+                        "offset": {
+                            "type": "string",
+                            "description": "The offset to start reading from",
+                            "enum": [
+                                "latest",
+                                "earliest"
+                            ]
+                        },
+                        "consumer_id": {
+                            "type": "integer",
```

Comment:
I think in general users shouldn't need to set a consumer name, since in arroyo we manage the horizontal scaling using low-level apis rather than high-level consumer group APIs.

It also looks like Iggy supports both numeric and textual consumers; I think many users would prefer to use names so perhaps it makes sense to support both types here by taking a string?